src/checkPassword.test.js

```diff
@@ -8,12 +8,35 @@ describe(`Function 'checkPassword':`, () => {
   });
 
   it(`should return boolean`, () => {
-
+    expect(typeof checkPassword('')).toBe('boolean');
   });
 
   it(`should return 'true' for the valid password with 8 characters`, () => {
+    expect(checkPassword('hellO.12')).toBe(true);
+  });
 
+  it(`should return 'false' for the password with 8 
+  characters without digits`, () => {
+    expect(checkPassword('hellO.oo')).toBe(false);
   });
 
-  // write more tests here
+  it(`should return 'false' for the password with 8 
+  characters without special characters`, () => {
+    expect(checkPassword('hellO123')).toBe(false);
+  });
+
+  it(`should return 'false' for the password with 8 
+  characters without uppercase letters`, () => {
```

Comment:
```suggestion
  it(`should return 'false' for the password without uppercase letters`, () => {
```
можна просто написати єдину умову, яка не дотримується